deploy/complete/helm-chart/mushop/charts/orders/templates/orders-deployment.yaml

```diff
@@ -20,6 +20,9 @@ spec:
         mockmode: {{ .Values.global.mock.service | quote }}
       annotations:
         sidecar.istio.io/rewriteAppHTTPProbers: "true"
+        prometheus.io/scrape: "true"
+        prometheus.io/path: /actuator/prometheus
```

Comment:
Could't you also use the actuators to do liveness and readiness probes?

Reply:
Yes - if fact it’s enabled. But the deployment template is not setup to use it, will add the liveness and readiness probes to the template